src/main/java/org/veupathdb/service/eda/ds/plugin/differentialabundance/DifferentialAbundanceVolcanoplotPlugin.java

```diff
@@ -0,0 +1,91 @@
+package org.veupathdb.service.eda.ds.plugin.differentialabundance;
+
+import org.gusdb.fgputil.validation.ValidationException;
+import org.veupathdb.service.eda.common.client.spec.StreamSpec;
+import org.veupathdb.service.eda.common.plugin.constraint.ConstraintSpec;
+import org.veupathdb.service.eda.common.plugin.constraint.DataElementSet;
+import org.veupathdb.service.eda.common.plugin.util.RServeClient;
+import org.veupathdb.service.eda.ds.Resources;
+import org.veupathdb.service.eda.ds.metadata.AppsMetadata;
+import org.veupathdb.service.eda.ds.plugin.AbstractPlugin;
+import org.veupathdb.service.eda.generated.model.*;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.veupathdb.service.eda.common.plugin.util.RServeClient.useRConnectionWithRemoteFiles;
+
+public class DifferentialAbundanceVolcanoplotPlugin extends AbstractPlugin<DifferentialAbundanceVolcanoplotPostRequest, DifferentialAbundanceVolcanoplotSpec, DifferentialAbundanceComputeConfig> {
+  
+  @Override
+  public String getDisplayName() {
+    return "Volcano plot";
+  }
+
+  @Override
+  public String getDescription() {
+    return "Display fold change vs. significance for a differential abundance analysis.";
+  }
+
+  @Override
+  public List<String> getProjects() {
+    return List.of(AppsMetadata.MICROBIOME_PROJECT);
+  }
+
+  @Override
+  protected ClassGroup getTypeParameterClasses() {
+    return new ClassGroup(DifferentialAbundanceVolcanoplotPostRequest.class, DifferentialAbundanceVolcanoplotSpec.class, DifferentialAbundanceComputeConfig.class);
+  }
+
+  // // I think we can remove this whole thing ?!? there are no variables for the user to select
```

Comment:
we need @ryanrdoherty .. bc i think outside of giving this viz a name and description this is just a pass-through endpoint for the stats endpoint from compute.

Reply:
I wrote up a utility (above) and used it (below) to do this and simplified the other required methods.  If we really think this is a common case maybe we can make a superclass for all the "no-config, stats-stream" viz plugins.  Future TBD.